site/en/docs/extensions/mv3/tut_puppeteer-testing/index.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Tutorial: Testing Chrome Extensions with Puppeteer"
+seoTitle: "Tutorial: Testing Chrome Extensions with Puppeteer"
+date: 2023-10-09
+description: How to write an automated test for Chrome Etensions in Puppeteer.
+---
+
+_For general advice on testing extensions, see
+[Automated testing for Chrome Extensions][automated-testing]._
```

Comment:
This feels like it should go after the intro to the tutorial and maybe an aside? Your call

Reply:
I think this is best so will leave for now, but open to changing it in the future.